spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -55,7 +55,7 @@ def preview_body(body)
   context 'with autocomplete results' do
     it 'returns collection of matches' do
       create(:version, :production)
-      create_list(:institution, 2, :start_like_harv)
+      create_list(:institution, 2, :start_like_harv, version_id: Version.current_production.id)
```

Comment:
Can you add this to the relevant factory instead of here and other places in this file?